eq-author/src/App/page/PropertiesPanel/GroupedAnswerProperties/index.js

```diff
@@ -72,17 +73,24 @@ const GroupContainer = styled.div`
   padding: 0.5em 0;
 `;
 
-export const UnitPropertiesStyled = styled(UnitProperties)`
-  ${({ hasUnitError }) => hasUnitError && `
-  border-color: ${colors.red};
-`}
-`;
+const filterUnitOptions = (options, query) =>
+  Object.values(options)
+    .filter(
+      x =>
+        x.unit.toLowerCase().includes(query.toLowerCase()) ||
+        x.abbreviation.toLowerCase().includes(query.toLowerCase())
```

Comment:
```suggestion
        x.unit.toLowerCase().includes(query.toLowerCase().trim()) ||
        x.abbreviation.toLowerCase().includes(query.toLowerCase().trim()) ||
        query.toLowerCase().trim().startsWith(x.unit.toLowerCase()) ||
        query.toLowerCase().trim().startsWith(x.abbreviation.toLowerCase())
```

What do you think of this? It makes it a bit more flexible in that whitespace at the start/end of user input is ignored - and it also means that when you type extra text after a matching fragment it still gives you suggestions. For instance, before this when you select an option and then you start editing it, you get a "No suggestions" message: 

<img width="360" alt="Screenshot 2020-12-10 at 09 11 20" src="https://user-images.githubusercontent.com/60441612/101752424-673fbd00-3ac9-11eb-8be4-b6e3987f3ac9.png">

After this change the option is still displayed which might be less confusing:

<img width="361" alt="Screenshot 2020-12-10 at 09 18 47" src="https://user-images.githubusercontent.com/60441612/101753288-5c395c80-3aca-11eb-994e-534139bcbd38.png">